metro.config.js

```diff
@@ -8,6 +8,44 @@
 
 const { getDefaultConfig } = require('expo/metro-config');
 const { mergeConfig } = require('@react-native/metro-config');
+const { lockdownSerializer } = require('@lavamoat/react-native-lockdown');
+
+// eslint-disable-next-line import/no-nodejs-modules
+const { parseArgs } = require('node:util');
+
+const parsedArgs = parseArgs({
+  options: {
+    platform: {
+      type: 'string',
+    },
+  },
+  allowPositionals: true,
+  strict: false,
+});
+
+// Apply SES Hermes shim on Android (Hermes) and Expo dev server by default
+let hermesRuntime = true;
+
+// Apply SES Vanilla shim on iOS (RN JSC)
+if (parsedArgs.values.platform === 'ios' || parsedArgs.positionals[0] === 'run:ios') {
+  // TODO: Remove once iOS upgraded to Hermes (i.e. both platforms on Hermes)
+  hermesRuntime = false;
+}
```

Comment:
```suggestion
// Apply SES Hermes shim on Android (Hermes) and Expo dev server by default
// and apply SES Vanilla shim on iOS (RN JSC)
// TODO: Remove once iOS upgraded to Hermes (i.e. both platforms on Hermes)
let hermesRuntime = parsedArgs.values.platform !== 'ios' && parsedArgs.positionals[0] !== 'run:ios';
```

this could be simplified, but nbd